File: others/2824-count_pairs.rb
# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer}
def count_pairs(nums, target)
    result = 0
    for i in 0...(nums.length)
        for j in (i + 1)...(nums.length)
            if (nums[i] + nums[j]) < target
                result = result + 1
            end
        end
    end
    return result
end

File: others/2824-countPairs.java
// https://leetcode.cn/problems/count-pairs-whose-sum-is-less-than-target/description/
class Solution {
  public int countPairs(List<Integer> nums, int target) {
    int result = 0;
    int n = nums.size();
    for (int i = 0; i < n; i++) {
      for (int j = i + 1; j < n; j++) {
        if ((nums.get(i) + nums.get(j)) < target) {
          result++;
        }
      }
    }
    return result;
  }
}
